bricepollock/canyoneer
# Rearchitect map-views to separate single and many-canyon maps

### Description
I think we over abstracted before with the hope to make switching map views easier. In reality, separating the APIs needed for rendering single and many canyons results in a much cleaner separation of concerns and will likely result in us more easily switching map viewers in the future.

Removed the "Show TOPO lines on map" button as we cannot support it with current implementation without moving to mapbox.

### Test Plan
* Load map and pan around
* Load an individual map to ensure its loading
